docs/DefineDifferentiability.md

```diff
@@ -0,0 +1,99 @@
+# A Short Guide on Defining Differentiability for ONNX operators
+
+The differentiability definition of an operator consists of several aspects.
```

Comment:
Add "The ONNX operator schema for each operator includes a Differentiability tag for each input and output. In this document, we explain the meaning of this tag and how to ensure that a correct tag-value is provided in a PR that proposes a new operator or updates an operator. Briefly, the tag identifies the set of differentiable inputs and differentiable outputs of an operator. The meaning of the tag is that the partial derivative of each differentiable output is defined with respect to each differentiable output."

Reply:
Ok.